x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -507,3 +506,7 @@ export const DEFAULT_ALERT_TAGS_VALUE = [
   i18n.FALSE_POSITIVE,
   i18n.FURTHER_INVESTIGATION_REQUIRED,
 ] as const;
+
+export const DEFAULT_SUPPRESSION_MISSING_FIELDS_STRATEGY = 'suppress' as const;
+
+export const MINIMUM_LICENSE_FOR_SUPPRESSION = 'platinum';
```

Comment:
It doesn't feel right to put domain-specific (and in this case rule-specific) constants in a super-generic `common/constants.ts` file that just accumulates random dust over time. We need to figure out where do we want to keep such code so it is inside subdomain folders.

Reply:
Extracted the constants. FYI this file consists mostly of domain-specific stuff, so it is probably worth reorganizing it.